best-cigars-guide/scripts/scripts.js

```diff
@@ -214,12 +214,22 @@ export function isInternal(path) {
   }
 }
 
+function addHreflang() {
```

Comment:
@tonyklapatch please add a description to this function. Should this go in header.js?

Reply:
Just added a description. scripts.js seemed more appropriate for this since it's where the lang attribute was already being set on the html element on line 234 and header.js doesn't add any meta info to the head element.